docs/UserGuide.adoc

```diff
@@ -56,47 +59,133 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 Format: `help`
 
+=== Clearing all entries : `avail`
+
+Lists all the available rooms of that type for the next NUM_DAYS days +
+Format: `avail r/ROOM TYPE num/NUM_DAYS`
+
+Lists all available rooms from start to end date. Dates have to be given in YYMMDD format
+
+Examples:
+
+* `avail r/Double Room num/5`
+* `avail r/Double Room ds/20180910 de/21080912`
+
+=== Listing all persons : `list`
+
+Shows a list of all room types and cost of room +
+Format: `list`
+
 === Adding a person: `add`
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a guest to the guest list for the purpose of assigning a room +
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]...`
+
+=== Adding a person: `addx`
+
+Adds a guest to the guest list NOT for the purpose of assigning a room +
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]...`
 
 [TIP]
-A person can have any number of tags (including 0)
+A guest can have any number of tags (including 0)
 
 Examples:
 
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
```

Comment:
Can update the above 2 to show the "add guest" format!

Reply:
wait do our guests even have tags

Reply:
we can include it since the feature's already been implemented
(e.g. tag people with disabilities)